Dockerfile

```diff
@@ -16,7 +20,7 @@ ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini ${TMP
 COPY docker/utils/startup.sh ${TMP_DIR}/bin/startup.sh
 RUN chmod +x -R ${TMP_DIR}/bin/
 
-FROM openjdk:11-jre-slim as app
+FROM openjdk:${JDK_TAG} as app
```

Comment:
Can we make that actually for the base image name, so we can eventually test with different distros? e.g. `BASE_IMAGE` or `JDK_IMAGE` or `APP_IMAGE` or anything.